packages/mobile/src/screens/SingleMovieStyle.tsx

```diff
@@ -0,0 +1,210 @@
+  height: 27,
+  width: 27,
+  position: 'absolute',
+  top: 10,
+  left: 15,
+});
+
+export const BackArrowImage = styled.Image({
+  height: 25,
+  width: 25,
+  position: 'absolute',
+  top: 10,
+  left: 15,
+});
+
+export const RowContainer = styled.View({
+  backgroundColor: BackgroundColor.BACKGROUNDCOLOR,
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  margin: 10,
+});
+
+export const ColumContainerLeft = styled.View({
+  backgroundColor: BackgroundColor.BACKGROUNDCOLOR,
+  flexDirection: 'column',
+});
+
+export const ColumContainerRight = styled.View({
+  backgroundColor: BackgroundColor.BACKGROUNDCOLOR,
+  flex: 2,
+});
+
+export const RatioContainer = styled.View({
+  backgroundColor: BackgroundColor.BACKGROUNDCOLOR,
+  flexDirection: 'row',
+  marginLeft: 70,
+  marginTop: 18,
+});
+
+export const StarsContainer = styled.View({
+  backgroundColor: BackgroundColor.BACKGROUNDCOLOR,
+  flexDirection: 'row',
+  alignSelf: 'flex-start',
+  marginLeft: 70,
+  marginTop: 18,
+});
+
+export const Star = styled.Image({
+  height: 25,
+  width: 25,
+  marginRight: 2,
+  shadowColor: "red",
+  shadowOffset: {
+	width: 0,
+	height: 6,
+},
+shadowOpacity: 0.39,
+shadowRadius: 8.30,
+elevation: 13,
+});
+
+export const Heart = styled.Image({
+  height: 40,
+  width: 40,
+  marginLeft: 30,
+});
+
+export const HeartRatio = styled.Text({
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.RATIO,
+  color: FontColors.HEADERS,
+  alignSelf: 'stretch',
+});
+
+export const VoteText = styled.Text({
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.MIDIUM,
+  fontWeight: FontWeight.BOLD,
+  color: FontColors.HEADERS,
+  marginTop: 5,
+  marginRight: 3,
+});
+
+export const MovieName = styled.Text({
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.MOVIENAME,
+  fontWeight: FontWeight.BOLD,
+  color: FontColors.HEADERS,
+});
+
+export const MovieKind = styled.Text({
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.MIDIUM,
+  fontWeight: FontWeight.BOLD,
+  color: FontColors.DESCRIPTION,
+});
+
+export const MovieTime = styled.Text({
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.VERYSMALL,
+  fontWeight: FontWeight.BOLD,
+  color: FontColors.DESCRIPTION,
+});
+
+export const Owerview = styled.Text({
+  marginTop: 15,
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.VERYSMALL,
+  fontWeight: FontWeight.BOLD,
+  color: FontColors.HEADERS,
+});
+
+export const Description = styled.Text({
+  marginLeft: 10,
+  fontFamily: FontFamily.ARIAL,
+  fontSize: FontSize.DESCRIPTION,
+  color: FontColors.DESCRIPTION,
+});
+
+export const PlayButton = styled.TouchableOpacity({
+  padding: 5,
+  height: 45,
+  width: 45, //The Width must be the same as the height
+  borderRadius: 90, //Then Make the Border Radius twice the size of width or Height
+  backgroundColor: 'transparent',
+  justifyContent: 'flex-start',
+  alignContent: 'space-around',
+  alignSelf: 'center',
+  position: 'absolute',
+  top: SIZE.HEIGHT-25,
```

Comment:
to jest błędy zapis który spowoduje niepoprawne wyswietlenie strzalki jesli slaider nie bedzie piewrszym elementem na tym widoku. Proponuje przeniesc PlayButton aby był w spólnym rodzicu ze zdjeciem i użyć bottom:-10 zamiast top